### ISSUE TITLE
omdb.set_default('tomatoes', True) not actually setting tomatoes=True for imdbid()

### ISSUE THREAD

**Issue by jplusc** [dgilland/omdb.py#20]
```
    omdb.set_default('apikey', OMDB_API_KEY)
    omdb.set_default('tomatoes', True)
    omdbresult = omdb.imdbid(a['imdbid'], timeout=10)
```
does *not* actually have fields such as `tomato_url` and the like returned.

however:
```
    omdb.set_default('apikey', OMDB_API_KEY)
    omdbresult = omdb.imdbid(a['imdbid'], tomatoes=True, timeout=10)
```
*does* return the normal empty tomatoes fields, but most importantly, the populated  `tomato_url`  field.

_actually, it looks like `.title()` suffers from this as well_

I am ok using the parameter tomatoes=True if I must, but the docs reference `omdb.set_default('tomatoes', True)`

Thanks!

**Comment by dgilland** [comment]
Definitely a bug. There's a workaround until it's fixed:

```python
omdb.set_default('tomatoes', 'true')  # string value for "true"
```

This would bypass the current bug that doesn't handle the conversion of defaults correctly when serializing the query parameters for the URL request.

**Comment by dgilland** [comment]
This should be fixed in `v0.10.1`: https://pypi.org/project/omdb/0.10.1/

If you have any other issues, let me know.

And thanks for reporting the bug! :+1:
